aws/resource_aws_emr_cluster.go

```diff
@@ -270,6 +270,11 @@ func resourceAwsEMRCluster() *schema.Resource {
 							DiffSuppressFunc: suppressEquivalentJsonDiffs,
 							ValidateFunc:     validation.ValidateJsonString,
 						},
+						"market": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
Please note that adding new `Required` arguments represents a breaking change and can only be accepted at the minimum during major provider version updates.

To keep backwards compatibility, I would recommend using `Optional: true` and `Computed: true` instead for these new `market` arguments.